Assets/HoloToolkit-Preview/QRTracker/Scripts/AttachToQRCode.cs

```diff
@@ -39,9 +38,11 @@ public string QRCodeData
                     {
                         qRCodeData = string.Empty;
                     }
-                    
+
+#if UNITY_EDITOR || UNITY_WSA
```

Comment:
what about `WINDOWS_UWP`?

Reply:
An [editor plugin](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/master/Assets/HoloToolkit-Preview/QRTracker/Plugins/QRCodesTrackerPlugin.Placeholder.dll) is provided, so it appears some parts of this (the ones not `WINDOWS_UWP`-specific) will work in the editor.

Of course, that's marked specifically as a placeholder, so...maybe not.

Reply:
No worries, I was just curious.